workflow-dockstore-metrics/workflow_dockstore_metrics/queries.py

```diff
@@ -0,0 +1,31 @@
+WORKFLOW_DATA_QUERY = '''
+WITH workflows_to_send as (
```

Comment:
Wonderful! Might be worth adding a comment explaining that this queries all Workflows that ended within the lookback period, regardless of when they started. This is an improvement on the original query that was used for the manual dumps so I think it is worth mentioning